MLS.Agent/CommandLine/CommandLineParser.cs

```diff
@@ -10,6 +10,7 @@
 using System.Threading.Tasks;
 using Clockwise;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.DotNet.Interactive;
```

Comment:
Why does this reference Interactive?

Reply:
IKernel is in that namespace

Reply:
.AddTransient<IKernel>(c => new CompositeKernel